### PR TITLE
[DEV-422] - Add tutorial list page layout

### PR DESCRIPTION
**Pull request by marcobottaro**
#### List of Changes
- Rename constants and file containing acronyms using camelCase
- Add layout to Tutorial list page
- Add content structure to handle Tutorial list for each product
- Change product type

#### Motivation and Context
The page was empty

#### How Has This Been Tested?
Manually

#### Screenshots (if appropriate):
image

#### Types of changes
- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/_contents/appIo/tutorialLists.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { appIo } from '@/_contents/appIo/appIo';
+import { appIoTutorialListsPath } from '@/_contents/appIo/tutorialListsPath';
+import { TutorialListsData } from '@/lib/types/tutorialListsData';
+import { Tutorial } from '@/lib/types/tutorialData';
+
+export const appIoTutorials: readonly Tutorial[] = [
+  {
+    title: 'Quali sono i possibili accordi di adesione all’app IO',
+    dateString: '2023-06-29T22:15:53.780Z',
+    path: '/app-io/tutorials/quale-accordo-di-adesione-scegliere',
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/appIo/tutorialLists.ts:10]:
```suggestion
    path: `${appIoTutorialListsPath.path}/quale-accordo-di-adesione-scegliere',
```

File: apps/nextjs-website/src/_contents/appIo/tutorialLists.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  {
+    title: 'Come inviare un messaggio con un avviso di pagamento',
+    dateString: '2023-06-29T22:15:53.780Z',
+    path: '/app-io/tutorials/come-spedire-un-avviso-di-pagamento-in-un-messaggio',
+    name: 'Come spedire un avviso di pagamento in un messaggio',
+  },
+  {
+    title: 'Come allegare documenti a un messaggio',
+    dateString: '2023-06-29T22:15:53.780Z',
+    path: '/app-io/tutorials/come-allegare-documenti-a-un-messaggio',
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/appIo/tutorialLists.ts:22]:
```suggestion
    path: `${appIoTutorialListsPath.path}/come-allegare-documenti-a-un-messaggio',
```

File: apps/nextjs-website/src/_contents/ioSign/tutorialLists.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { ioSign } from '@/_contents/ioSign/ioSign';
+import { ioSignTutorialListsPath } from '@/_contents/ioSign/tutorialListsPath';
+import { TutorialListsData } from '@/lib/types/tutorialListsData';
+import { Tutorial } from '@/lib/types/tutorialData';
+
+export const ioSignTutorials: readonly Tutorial[] = [
+  {
+    title: 'Come creare e preparare il documento da firmare digitalmente',
+    dateString: '2023-06-29T22:15:53.780Z',
+    path: '/io-sign/tutorials/1',
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/ioSign/tutorialLists.ts:10]:
```suggestion
    path: `${ioSignTutorialListsPath.path}/1`,
```

File: apps/nextjs-website/src/_contents/ioSign/tutorialLists.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    name: 'tutorial 1',
+  },
+  {
+    title: 'Come creare il Dossier per la richiesta di firma',
+    dateString: '2023-06-29T22:15:53.780Z',
+    path: '/io-sign/tutorials/2',
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/ioSign/tutorialLists.ts:16]:
```suggestion
    path: `${ioSignTutorialListsPath.path}/2`,
```

File: apps/nextjs-website/src/_contents/ioSign/tutorialLists.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    name: 'tutorial 2',
+  },
+  {
+    title: 'Come effettuare l’upload dei documenti',
+    dateString: '2023-06-29T22:15:53.780Z',
+    path: '/io-sign/tutorials/3',
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/ioSign/tutorialLists.ts:22]:
```suggestion
    path: `${ioSignTutorialListsPath.path}/3`,
```

File: apps/nextjs-website/src/_contents/pagoPa/tutorialLists.ts [hunk 6]
```diff
@@ -0,0 +1,35 @@
+import { pagoPa } from '@/_contents/pagoPa/pagoPa';
+import { pagoPaTutorialListsPath } from '@/_contents/pagoPa/tutorialListsPath';
+import { TutorialListsData } from '@/lib/types/tutorialListsData';
+import { Tutorial } from '@/lib/types/tutorialData';
+
+export const pagoPaTutorials: readonly Tutorial[] = [
+  {
+    title: 'Come richiedere pagamenti che contengono marca da bollo digitale',
+    dateString: '2023-06-29T22:15:53.780Z',
+    path: '/pago-pa/tutorials/come-richiedere-pagamenti-che-contengono-marca-da-bollo-digitale',
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/pagoPa/tutorialLists.ts:10]:
```suggestion
    path: `${pagoPaTutorialListsPath.path}/come-richiedere-pagamenti-che-contengono-marca-da-bollo-digitale`,
```

File: apps/nextjs-website/src/_contents/pagoPa/tutorialLists.ts [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    name: 'Come richiedere pagamenti che contengono marca da bollo digitale',
+  },
+  {
+    title: 'Come avviare un esercizio come Ente Creditore su pagoPA',
+    dateString: '2023-06-29T22:15:53.780Z',
+    path: '/pago-pa/tutorials/come-avviare-un-esercizio-come-ente-creditore-su-pagopa',
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/pagoPa/tutorialLists.ts:16]:
```suggestion
    path: `${pagoPaTutorialListsPath.path}/come-avviare-un-esercizio-come-ente-creditore-su-pagopa`,
```

File: apps/nextjs-website/src/_contents/pagoPa/tutorialLists.ts [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+    name: 'Come avviare un esercizio come Ente Creditore su pagoPA',
+  },
+  {
+    title: 'Come stampare un avviso di pagamento in formato PDF',
+    dateString: '2023-06-29T22:15:53.780Z',
+    path: '/pago-pa/tutorials/come-stampare-un-avviso-di-pagamento-in-formato-pdf',
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/pagoPa/tutorialLists.ts:22]:
```suggestion
    path: `${pagoPaTutorialListsPath.path}/come-stampare-un-avviso-di-pagamento-in-formato-pdf`,
```

File: apps/nextjs-website/src/pages/[productSlug]/tutorials.tsx [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+          theme={palette.mode}
+        />
+      )}
+      {product.subpaths.tutorials && tutorials && (
+        <Box>
+          <Newsroom
+            items={tutorials.map((tutorial) => ({
+              title: tutorial.title,
+              date: {
+                date: new Date(),
```
Comment (datalek) [inline apps/nextjs-website/src/pages/[productSlug]/tutorials.tsx:78]:
```suggestion
                date: new Date(tutorial.dateString),
```

**Review by datalek — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/pages/[productSlug]/tutorials.tsx [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+import {
+  getTutorialLists,
+  getTutorialListsPaths,
+  getProducts,
+} from '@/lib/api';
+import { Abstract } from '@/editorialComponents/Abstract/Abstract';
+import { Box, useTheme } from '@mui/material';
+import Layout, { LayoutProps } from '@/components/organisms/Layout/Layout';
+import { Tutorial } from '@/lib/types/tutorialData';
+import { Newsroom } from '@pagopa/pagopa-editorial-components';
```
Comment (datalek) [inline apps/nextjs-website/src/pages/[productSlug]/tutorials.tsx:12]:
```suggestion
import Newsroom from '@/editorialComponents/Newsroom/Newsroom';
```

Comment (marcobottaro) [reply apps/nextjs-website/src/pages/[productSlug]/tutorials.tsx:12]: Direi proprio di sì!!!

**Review by datalek — APPROVED** [review]

**Pull request merged**